common/djangoapps/static_replace/models.py

```diff
@@ -0,0 +1,25 @@
+"""
+Models for static_replace
+"""
+
+from django.db.models.fields import TextField
+from config_models.models import ConfigurationModel
+
+
+class AssetBaseUrlConfig(ConfigurationModel):
+    """Configuration for the base URL used for static assets."""
+    base_url = TextField(
+        blank=True,
+        help_text="The alternative hostname to serve static assets from.  Should be in the form of hostname[:port]."
+    )
+
```

Comment:
On that note, since we will have to set something for expires (I think we just leave it blank now, but we're initially going to put it at an hour), could you please add that config option to `common/djangoapps/contentserver` and use the value in the content responses?